crates/bindings-csharp/Codegen/Module.cs

```diff
@@ -209,15 +301,15 @@ public override Scope.Extensions ToExtensions()
                             ",\n",
                             Members
                             // Important: the position must be stored here, before filtering.
-                            .Select((col, pos) => (col, pos))
-                            .Where(pair => pair.col.Attrs != ColumnAttrs.UnSet)
+                            .Select((col, pos) => (col, pos, col.GetAttrs(v.Name)))
+                            .Where(tuple => tuple.Item3 != ColumnAttrs.UnSet)
                             .Select(pair =>
                                 $$"""
                                 new (
                                     nameof({{ShortName}}),
                                     {{pair.pos}},
                                     nameof({{pair.col.Name}}),
-                                    SpacetimeDB.ColumnAttrs.{{pair.col.Attrs}}
+                                    (SpacetimeDB.ColumnAttrs){{(int)pair.Item3}}
```

Comment:
Let's keep the typechecked enum instead, like it did before by stringifying the enum's member name.

Reply:
Ah yeah now there's separate attributes, like [AutoInc] and [PrimaryKey], so either its a cast to int here or emitting the full bitwise or expression.

Reply:
Figured (cast)15 is better than (ColumnAttrs.PrimaryKey | ColumnAttrs.Unique) from a codegen complexity vs effect perspective.

Reply:
> is better than (ColumnAttrs.PrimaryKey | ColumnAttrs.Unique) from a codegen complexity vs effect perspective

We actually have names for their combinations too, so this shouldn't be a problem. At least before you could also specify `[Column(ColumnAttrs.PrimaryKey | ColumnAttrs.Unique)]` and it would get stringified as `PrimaryKeyAuto` automatically.

I'm probably going to change this.